### PR TITLE
Updated README.md with mobile solutions

### LINKED ISSUE #5: Make an equivalent solution for smart phones

### PR DESCRIPTION
**Pull request by sahil-shubham**
Added step wise instructions for mobile users to be able to use this extension; fixes #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,27 @@ A chrome extension that you have to setup just one time, to automatically login
 
 <img src="demo.png">
 
+## How to use this extension on Mobile
+
+Mobile users can also use erp auto login using browsers like **Yandex** and **Kiwi**. The steps to follow for the browsers respectively are:- 
+
+- ### Yandex Browser
+
+  1. Dowload Yandex Browser from Play Store
```
Comment (thealphadollar) [inline README.md:14]: Nitpicking: spelling of download

File: README.md [hunk 2]
```diff
@@ -5,6 +5,27 @@ A chrome extension that you have to setup just one time, to automatically login
...
+  2. Head over to [Erp auto login on chrome Web Store](https://chrome.google.com/webstore/detail/iitkgp-erp-auto-login/gdickphilmdekcbmpjmbnbikchaecbdk)
+  3. Now click on Desktop version 
+  4. Now click on **add to chrome** to add the extension
+  5. Go to extensions and select erp auto login
+  6. Enter your details
+
+- ### Kiwi Browser
+  
+  1. Dowload Kiwi Browser from Play Store
```
Comment (thealphadollar) [inline README.md:23]: Nitpicking: spelling of download

**Review by thealphadollar — CHANGES_REQUESTED** [review]
This is super cool! And I really like how you have gone a step further to add detailed instructions for both browsers. Thanks a lot for the efforts Sahil ❤️

I've suggested two minor changes 😀 Please fix them as your schedule permits.

File: README.md (hunk 2 above)
Comment (sahil-shubham) [reply README.md]: done, thank you.

**Comment by sahil-shubham** [comment]
Should I also add screenshots for few of the steps?

**Comment by thealphadollar** [comment]
Yeah, that would be fabulous if it doesn't take much of your time.

File: README.md [hunk 3]
```diff
@@ -1,10 +1,40 @@
...
+  1. Download Yandex Browser from Play Store
+  2. Head over to [Erp auto login on chrome Web Store](https://chrome.google.com/webstore/detail/iitkgp-erp-auto-login/gdickphilmdekcbmpjmbnbikchaecbdk)
+  3. Now click on Desktop version   
+
+    [yandex-desktop-version](instruction_imgaes/yandex_desktop.jpg)
+
+  4. Now click on **add to chrome** to add the extension
+  5. Go to extensions and select erp auto login
+
+    [yandex-extensions](instruction_imgaes/yandex_extensions.jpg)
```
Comment (thealphadollar) [inline README.md:23]:
```suggestion
    [yandex-extensions](instruction_images/yandex_extensions.jpg)
```

File: README.md [hunk 4]
```diff
@@ -1,10 +1,40 @@
...
+
+Mobile users can also use erp auto login using browsers like **Yandex** and **Kiwi**. The steps to follow for the browsers respectively are:-
+
+- ### Yandex Browser
+
+  1. Download Yandex Browser from Play Store
+  2. Head over to [Erp auto login on chrome Web Store](https://chrome.google.com/webstore/detail/iitkgp-erp-auto-login/gdickphilmdekcbmpjmbnbikchaecbdk)
+  3. Now click on Desktop version   
+
+    [yandex-desktop-version](instruction_imgaes/yandex_desktop.jpg)
```
Comment (thealphadollar) [inline README.md:18]:
```suggestion
    [yandex-desktop-version](instruction_images/yandex_desktop.jpg)
```

File: README.md [hunk 5]
```diff
@@ -1,10 +1,40 @@
...
+  6. Enter your details
+
+- ### Kiwi Browser
+
+  1. Download Kiwi Browser from Play Store
+  2. Head over to [Erp auto login on chrome Web Store](https://chrome.google.com/webstore/detail/iitkgp-erp-auto-login/gdickphilmdekcbmpjmbnbikchaecbdk)
+  3. Now click on **add to chrome** to add the extension
+  4. Click on three-dot menu on top right, scroll below and select **erp auto login**
+
+    [kiwi-extensions](instruction_imgaes/kiwi_extensions.jpg)
```
Comment (thealphadollar) [inline README.md:34]:
```suggestion
    [kiwi-extensions](instruction_images/kiwi_extensions.jpg)
```

**Review by thealphadollar — CHANGES_REQUESTED** [review]
This is super cool - I liked how you have emphasized the important part of the image by marking it... that is ingenious!

There are three trivial spelling issues, I've added suggestions for fixing them :rocket:

**Comment by sahil-shubham** [comment]
@thealphadollar Thank you, I started fixing these right when I pushed the commit. I would have to decrease the amount of spelling mistakes I make :sweat_smile: 

I decreased the width of the images as they looked really big. Plus, I also added a link to CONTRIBUTING.md, it has nothing to do with this issue but it makes the README better.

**Review by thealphadollar — APPROVED** [review]
This looks amazing to me, and ready to ship! Thanks a lot Sahil for the amazing contributions, and going one step extra, I didn't expect anything less :star2:

**Pull request merged**